.github/workflows/check_release_tag.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+"""Check that the GitHub release tag matches the package version."""
+import argparse
+import json
+
+if __name__ == '__main__':
```

Comment:
I take it you use `argparse` here instead of `click` because the action that calls it doesn't have `click` installed?

Reply:
yeh exactly, for something so simple I didn't want the extra complication of a click dependency